### ISSUE TITLE
Exception in thread "Thread-2" java.lang.IllegalStateException: Neither Jetty ALPN nor OpenSSL via netty-tcnative were properly configured.

### ISSUE THREAD

**Issue by russmac** [brianfrankcooper/YCSB#861]
Following all the instructions for [bigtable](https://github.com/brianfrankcooper/YCSB/tree/master/googlebigtable) and ycsb main readme I'm encountering an error trying to load a workload.

I've gotten the correct jar of alpn-boot to match my JRE version. alpn-boot-8.1.9.v20160720.jar 

And am passing it to the jre  -Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext

I've followed all the steps in the linked guides to get the hbase client up and created the table, I used 

It looks like it works as ycsb loads.  "Thread-2" makes it sound like something happens before the error.

I've found one other issue reported with a similar error: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/issues/912

My JRE is: 1.8.0u102

root@biginstance-1:~# $JAVA_HOME/bin/java -version
openjdk version "1.8.0_102"
OpenJDK Runtime Environment (build 1.8.0_102-8u102-b14.1-1~bpo8+1-b14)
OpenJDK 64-Bit Server VM (build 25.102-b14, mixed mode)
	
The alpn-boot version is : 8.1.9.v20160720

```
root@biginstance-1:~/ycsb-0.11.0# bin/ycsb load googlebigtable -p columnfamily=cf -p google.bigtable.project.id=benchmark-nosql -p google.bigtable.cluster.name=bigtable-us-east-1-cluster -p google.bigtable.zone.name=us-east-1b -p google.bigtable.auth.service.account.enable=true -p debug=true -jvm-args='-Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext' -P workloads/workloadb

/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -cp /root/ycsb-0.11.0/googlebigtable-binding/conf:/root/ycsb-0.11.0/conf:/root/ycsb-0.11.0/lib/core-0.11.0.jar:/root/ycsb-0.11.0/lib/jackson-mapper-asl-1.9.4.jar:/root/ycsb-0.11.0/lib/HdrHistogram-2.1.4.jar:/root/ycsb-0.11.0/lib/jackson-core-asl-1.9.4.jar:/root/ycsb-0.11.0/lib/htrace-core4-4.1.0-incubating.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hbase-protocol-1.0.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-yarn-common-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hbase-client-1.0.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-net-3.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/googlebigtable-binding-0.11.0.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/snappy-java-1.0.4.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-compress-1.4.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/jackson-mapper-asl-1.9.4.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-common-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/netty-3.6.2.Final.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/paranamer-2.3.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/activation-1.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/httpclient-4.2.5.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/log4j-1.2.17.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/xmlenc-0.52.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/findbugs-annotations-1.3.9-1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-mapreduce-client-core-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/jsr305-1.3.9.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-httpclient-3.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-beanutils-1.7.0.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-annotations-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-auth-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/jackson-core-asl-1.9.4.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hbase-common-1.0.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-logging-1.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/stax-api-1.0-2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-digester-1.8.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/hadoop-yarn-api-2.5.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-configuration-1.6.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/httpcore-4.2.4.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/jsch-0.1.42.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/avro-1.7.4.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-el-1.0.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-collections-3.2.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-beanutils-core-1.8.0.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/bigtable-hbase-1.0-0.2.3.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/slf4j-api-1.6.4.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-lang-2.6.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-math3-3.1.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/jaxb-api-2.2.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/commons-cli-1.2.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/guava-12.0.1.jar:/root/ycsb-0.11.0/googlebigtable-binding/lib/xz-1.0.jar com.yahoo.ycsb.Client -db com.yahoo.ycsb.db.GoogleBigtableClient -p columnfamily=cf -p google.bigtable.project.id=benchmark-nosql -p google.bigtable.cluster.name=bigtable-us-east-1-cluster -p google.bigtable.zone.name=us-east-1b -p google.bigtable.auth.service.account.enable=true -p google.bigtable.auth.json.keyfile=/root/.config/gcloud/application_default_credentials.json -p jvm-args=-javaagent:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/jetty-alpn-agent-2.0.4 -Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/alpn-boot-8.1.9.v20160720.jar -p debug=true -P workloads/workloadb -load
YCSB Client 0.11.0
Command line: -db com.yahoo.ycsb.db.GoogleBigtableClient -p columnfamily=cf -p google.bigtable.project.id=benchmark-nosql -p google.bigtable.cluster.name=bigtable-us-east-1-cluster -p google.bigtable.zone.name=us-east-1b -p google.bigtable.auth.service.account.enable=true -p google.bigtable.auth.json.keyfile=/root/.config/gcloud/application_default_credentials.json -p jvm-args=-javaagent:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/jetty-alpn-agent-2.0.4 -Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/alpn-boot-8.1.9.v20160720.jar -p debug=true -P workloads/workloadb -load
Loading workload...
log4j:WARN No appenders could be found for logger (org.apache.htrace.core.Tracer).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Starting test.
Running Google Bigtable with Proto API.
Exception in thread "Thread-2" java.lang.IllegalStateException: Neither Jetty ALPN nor OpenSSL via netty-tcnative were properly configured.
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:255)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:230)
	at com.yahoo.ycsb.db.GoogleBigtableClient.init(GoogleBigtableClient.java:133)
	at com.yahoo.ycsb.DBWrapper.init(DBWrapper.java:99)
	at com.yahoo.ycsb.ClientThread.run(Client.java:418)
	at java.lang.Thread.run(Thread.java:745)
[OVERALL], RunTime(ms), 77.0
[OVERALL], Throughput(ops/sec), 0.0
[TOTAL_GCS_PS_Scavenge], Count, 1.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 4.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 5.194805194805195
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
[TOTAL_GC_TIME_%_PS_MarkSweep], Time(%), 0.0
[TOTAL_GCs], Count, 1.0
[TOTAL_GC_TIME], Time(ms), 4.0
[TOTAL_GC_TIME_%], Time(%), 5.194805194805195
```

Output from hbase check config
```
root@biginstance-1:~/hbase-1.2.1# ./bin/hbase com.google.cloud.bigtable.hbase.CheckConfig
User Agent: bigtable-hbase-1.2-0.9.3,1.8
Project ID: benchmark-nosql
Instance Id: bigtable-us-east-1
Instance admin host: bigtableadmin.googleapis.com
Table admin host: bigtableadmin.googleapis.com
Data host: bigtable.googleapis.com
Attempting credential refresh...
HBase Connection Class = com.google.cloud.bigtable.hbase1_2.BigtableConnection (OK)
Opening table admin connection...
2016-10-10 08:38:45,526 WARN  [main] util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2016-10-10 08:38:45,781 INFO  [main] grpc.BigtableSession: Opening connection for projectId benchmark-nosql, instanceId bigtable-us-east-1, on data host bigtable.googleapis.com, table admin host bigtableadmin.googleapis.com.
2016-10-10 08:38:45,784 INFO  [main] grpc.BigtableSession: Bigtable options: BigtableOptions{dataHost=bigtable.googleapis.com, tableAdminHost=bigtableadmin.googleapis.com, instanceAdminHost=bigtableadmin.googleapis.com, projectId=benchmark-nosql, instanceId=bigtable-us-east-1, userAgent=bigtable-hbase-1.2-0.9.3,1.8, credentialType=DefaultCredentials, port=443, dataChannelCount=5, retryOptions=RetryOptions{retriesEnabled=true, allowRetriesWithoutTimestamp=false, statusToRetryOn=[ABORTED, INTERNAL, UNAVAILABLE, DEADLINE_EXCEEDED], initialBackoffMillis=5, maxElaspedBackoffMillis=60000, backoffMultiplier=2.0, streamingBufferSize=60, readPartialRowTimeoutMillis=60000, maxScanTimeoutRetries=3}, bulkOptions=BulkOptions{asyncMutatorCount=2, useBulkApi=true, bulkMaxKeyCount=25, bulkMaxRequestSize=1048576, maxInflightRpcs=250, maxMemory=213005107}, callOptionsConfig=CallOptionsConfig{useTimeout=false, shortRpcTimeoutMs=60000, longRpcTimeoutMs=60000}, usePlaintextNegotiation=false}.
2016-10-10 08:38:46,101 INFO  [BigtableSession-startup-0] grpc.BigtableSession: SslContext: gRPC is using the OpenSSL provider (tcnactive jar - Open Ssl version: BoringSSL)
2016-10-10 08:38:46,183 INFO  [Credentials-Refresh-0] io.RefreshingOAuth2CredentialsInterceptor: Refreshing the OAuth token
Oct 10, 2016 8:38:46 AM com.google.bigtable.repackaged.io.grpc.internal.ManagedChannelImpl <init>
INFO: [ManagedChannelImpl@139982de] Created with target directaddress:///bigtable.googleapis.com/173.194.214.95:443
Oct 10, 2016 8:38:46 AM com.google.bigtable.repackaged.io.grpc.internal.ManagedChannelImpl <init>
INFO: [ManagedChannelImpl@ba54932] Created with target directaddress:///bigtableadmin.googleapis.com/173.194.214.95:443
Tables in cluster bigtable-us-east-1:
usertable
Closing connection...
Oct 10, 2016 8:38:46 AM com.google.bigtable.repackaged.io.grpc.internal.ManagedChannelImpl maybeTerminateChannel
INFO: [ManagedChannelImpl@139982de] Terminated
Oct 10, 2016 8:38:46 AM com.google.bigtable.repackaged.io.grpc.internal.ManagedChannelImpl maybeTerminateChannel
INFO: [ManagedChannelImpl@ba54932] Terminated
```

**Comment by risdenk** [comment]
It looks like you have `-p jvm-args=...` when it should be `-jvm-args=...`

`-jvm-args='-javaagent:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/jetty-alpn-agent-2.0.4 -Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext'`

**Comment by risdenk** [comment]
You probably also have to move the `-jvm-args` argument before the `load` argument.

**Comment by russmac** [comment]
```
An argument value without corresponding argument specifier (e.g., -p, -s) was found.
```
 It looks like it needs it? (the -p)
The jvm args is after load in the documentation.

I've edited original post to show my most recent attempt. I tried using the alpn agent as well as specifying its class path directly. (alpn) the documentation does not seem correct.

**Comment by risdenk** [comment]
Add the dash (-) in front of `jvm-args` like so `-jvm-args`.

**Comment by russmac** [comment]
I still get the same issue, I would of tried 20-80 different invocations at this point including all of the above. Nothing gets rid of the "IllegalStateException".

```
bin/ycsb load googlebigtable -p columnfamily=cf -p google.bigtable.project.id=benchmark-nosql -p google.bigtable.cluster.name=bigtable-us-east-1-cluster -p google.bigtable.zone.name=us-east-1b -p google.bigtable.auth.service.account.enable=true -p debug=true -jvm-args='-Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext' -P workloads/workloadb

# I can see the arg being prepended to java
/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -Xbootclasspath/p:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext 
```

**Comment by russmac** [comment]
I enabled log4j properly and got the following debug output
```
sampler.classes = ; loaded no samplers
span.receiver.classes = ; loaded no span receivers
Project ID benchmark-nosql
Zone us-east-1b
Cluster bigtable-us-east-1-cluster
API Data endpoint host bigtable.googleapis.com.
Table Admin endpoint host bigtabletableadmin.googleapis.com.
Cluster Admin endpoint host bigtableclusteradmin.googleapis.com.
Using service accounts
Using json keyfile: service_key.json
gRPC retries enabled: true
gRPC retry on deadline exceeded enabled: true
gRPC retry maxElapsedBackoffMillis: 60000
gRPC read partial row timeout (millis): 60000
gRPC read buffer size (count): 60
gRPC read batch size (count): 30
Connection Configuration: projectId: benchmark-nosql, zoneId: us-east-1b, clusterId: bigtable-us-east-1-cluster, data host bigtable.googleapis.com, table admin host bigtabletableadmin.googleapis.com, cluster admin host bigtableclusteradmin.googleapis.com.
Opening connection for projectId benchmark-nosql, zoneId us-east-1b, clusterId bigtable-us-east-1-cluster, on data host bigtable.googleapis.com, table admin host bigtabletableadmin.googleapis.com.
Using Log4J as the default logging framework
-Dio.netty.tmpdir: /tmp (java.io.tmpdir)
-Dio.netty.native.workdir: /tmp (io.netty.tmpdir)
Failed to load netty-tcnative; OpenSslEngine will be unavailable. See http://netty.io/wiki/forked-tomcat-native.html for more information.
java.lang.UnsatisfiedLinkError: org.apache.tomcat.jni.Library.version(I)I
	at org.apache.tomcat.jni.Library.version(Native Method)
	at org.apache.tomcat.jni.Library.initialize(Library.java:176)
	at com.google.bigtable.repackaged.io.netty.handler.ssl.OpenSsl.<clinit>(OpenSsl.java:59)
	at com.google.cloud.bigtable.grpc.BigtableSession.isAlpnProviderEnabled(BigtableSession.java:126)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:252)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:230)
	at com.yahoo.ycsb.db.GoogleBigtableClient.init(GoogleBigtableClient.java:133)
	at com.yahoo.ycsb.DBWrapper.init(DBWrapper.java:99)
	at com.yahoo.ycsb.ClientThread.run(Client.java:418)
	at java.lang.Thread.run(Thread.java:745)
OpenSSL available: false
Jetty ALPN available: false
OpenSSL available: false
Jetty ALPN available: false
Neither Jetty ALPN nor OpenSSL are available. OpenSSL unavailability cause:
java.lang.UnsatisfiedLinkError: org.apache.tomcat.jni.Library.version(I)I
```

**Comment by russmac** [comment]
I get in the same issue in the hbase10 binding

```
Starting test.
com.yahoo.ycsb.DBException: java.io.IOException: java.lang.reflect.InvocationTargetException
	at com.yahoo.ycsb.db.HBaseClient10.init(HBaseClient10.java:153)
	at com.yahoo.ycsb.DBWrapper.init(DBWrapper.java:99)
	at com.yahoo.ycsb.ClientThread.run(Client.java:418)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: java.lang.reflect.InvocationTargetException
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:240)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:218)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:119)
	at com.yahoo.ycsb.db.HBaseClient10.init(HBaseClient10.java:149)
	... 3 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:238)
	... 6 more
Caused by: java.lang.IllegalArgumentException: Jetty ALPN/NPN has not been properly configured.
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.selectApplicationProtocolConfig(GrpcSslContexts.java:153)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:130)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:119)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.forClient(GrpcSslContexts.java:90)
	at com.google.cloud.bigtable.grpc.BigtableSession.createSslContext(BigtableSession.java:108)
	at com.google.cloud.bigtable.grpc.BigtableSession.createNettyChannel(BigtableSession.java:513)
	at com.google.cloud.bigtable.grpc.BigtableSession.createNettyChannel(BigtableSession.java:502)
	at com.google.cloud.bigtable.grpc.BigtableSession$6.create(BigtableSession.java:485)
	at com.google.cloud.bigtable.grpc.io.ChannelPool.<init>(ChannelPool.java:238)
	at com.google.cloud.bigtable.grpc.BigtableSession.createChannelPool(BigtableSession.java:481)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.<init>(BigtableClusterUtilities.java:89)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.forAllInstances(BigtableClusterUtilities.java:72)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.lookupInstanceId(BigtableClusterUtilities.java:253)
	at com.google.cloud.bigtable.grpc.BigtableSession.resolveLegacyOptions(BigtableSession.java:305)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:239)
	at org.apache.hadoop.hbase.client.AbstractBigtableConnection.<init>(AbstractBigtableConnection.java:137)
	at com.google.cloud.bigtable.hbase1_0.BigtableConnection.<init>(BigtableConnection.java:56)
	... 11 more
com.yahoo.ycsb.DBException: java.io.IOException: java.lang.reflect.InvocationTargetException
	at com.yahoo.ycsb.db.HBaseClient10.init(HBaseClient10.java:153)
	at com.yahoo.ycsb.DBWrapper.init(DBWrapper.java:99)
	at com.yahoo.ycsb.ClientThread.run(Client.java:418)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: java.lang.reflect.InvocationTargetException
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:240)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:218)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:119)
	at com.yahoo.ycsb.db.HBaseClient10.init(HBaseClient10.java:149)
	... 3 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:238)
	... 6 more
Caused by: java.lang.IllegalArgumentException: Jetty ALPN/NPN has not been properly configured.
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.selectApplicationProtocolConfig(GrpcSslContexts.java:153)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:130)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:119)
	at com.google.bigtable.repackaged.io.grpc.netty.GrpcSslContexts.forClient(GrpcSslContexts.java:90)
	at com.google.cloud.bigtable.grpc.BigtableSession.createSslContext(BigtableSession.java:108)
	at com.google.cloud.bigtable.grpc.BigtableSession.createNettyChannel(BigtableSession.java:513)
	at com.google.cloud.bigtable.grpc.BigtableSession.createNettyChannel(BigtableSession.java:502)
	at com.google.cloud.bigtable.grpc.BigtableSession$6.create(BigtableSession.java:485)
	at com.google.cloud.bigtable.grpc.io.ChannelPool.<init>(ChannelPool.java:238)
	at com.google.cloud.bigtable.grpc.BigtableSession.createChannelPool(BigtableSession.java:481)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.<init>(BigtableClusterUtilities.java:89)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.forAllInstances(BigtableClusterUtilities.java:72)
	at com.google.cloud.bigtable.grpc.BigtableClusterUtilities.lookupInstanceId(BigtableClusterUtilities.java:253)
	at com.google.cloud.bigtable.grpc.BigtableSession.resolveLegacyOptions(BigtableSession.java:305)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:239)
	at org.apache.hadoop.hbase.client.AbstractBigtableConnection.<init>(AbstractBigtableConnection.java:137)
	at com.google.cloud.bigtable.hbase1_0.BigtableConnection.<init>(BigtableConnection.java:56)
	... 11 more

```

**Comment by Serapke** [comment]
Did you find out why was this failing?

**Comment by russmac** [comment]
I never found out @Serapke .

**Comment by mbrukman** [comment]
I think this should be fixed when issue #1048 is closed (there's a PR sent for review).
